src/components/SearchBox/SearchBox.tsx

```diff
@@ -20,9 +20,10 @@ const TopPanel = styled.div`
   padding: 8px;
```

Comment:
I would expect the red stripe not to go stretch to the left. Would you agree with that?

Like this:
![Image](https://github.com/user-attachments/assets/c220ac89-84cb-4f5d-b99b-3044cea383b5)